_posts/2016-9-26-ml-theory-pt2.md

```diff
@@ -70,7 +70,7 @@ If you noticed, all our analysis up till now was focusing on a **single** hypoth
 
  In order for the entire hypothesis space to have a generalization gap bigger than $\epsilon$, at least one of its hypothesis: $h_1$ **or** $h_2$ **or** $h_3$ **or** ... etc should have. This can be expressed formally by stating that:
 
- $$\mathbb{P}\left[\sup_{h \in \mathcal{H}}|R(h) - R_\text{emp}(h)| > \epsilon\right] = \mathbb{P}\left[\bigcup_{h \in \mathcal{H}} |R(h) - R_\text{emp}(h)| > \epsilon\right]$$
+ $$\mathbb{P}\left[\sup_{h \in \mathcal{H}}|R(h) - R_\text{emp}(h)| > \epsilon\right] \leq \mathbb{P}\left[\bigcup_{h \in \mathcal{H}} |R(h) - R_\text{emp}(h)| > \epsilon\right]$$
```

Comment:
The probability of the whole space having a generalization gap bigger than epsilon _**is**_ the probability of at least one of the hypotheses in the space having such gap. So the equal sign here is correct, the less-than-or-equal appears in the next equation when we bound the probability of the union by the sum of its individual probabilities.